tools/scoring/score/dimensions/raw_field_selection.py

```diff
@@ -29,14 +29,36 @@ class RawFieldSelection(Dimension):
   def __init__(self, *, translations: TranslationsDict):
     super().__init__(translations=translations)
 
-    solution_fields = set(
-        map(lambda item: item[1].raw_field_name, translations[SOLUTION]))
-    proposed_fields = set(
-        map(lambda item: item[1].raw_field_name, translations[PROPOSED]))
+    # Combine translations for all devices within the dictionary
+    solution_condensed = [
+        matched_translations[SOLUTION]
```

Comment:
this could have a helper function foo(stuff) that we call twice:
foo(SOLUTION)
foo(PROPOSED)

it is the same code repeated, only the input is changing.
what do you think?

Reply:
I don't have a problem with that other than the fact that it adds an extra assignment for a pretty simply expression — but it does add some degree of protection against dumb mistakes. Will do.